Fix wrong dependencies

AsterITA opened the pull request:
I noticed that the tsx dep wasn't always included and also moved some types/node to devDeps

taglioni-r reviewed (commented):
I think that the packages that didn't have the `tsx` dependency are the ones that do not have their own `start` procedure (such as `commons`). If you check a start procedure (like the one in `catalog-process`), that's where `tsx` is used. So there is no need to add an unused dependency.

AsterITA commented:
> I think that the packages that didn't have the `tsx` dependency are the ones that do not have their own `start` procedure (such as `commons`). If you check a start procedure (like the one in `catalog-process`), that's where `tsx` is used. So there is no need to add an unused dependency.

You're right, I removed the one that weren't needed and I also removed the start script from `commons` and `models`

ecamellini commented:
@AsterITA could you use this PR also to remove the "uuid" dependency from the BFF? It's unused and shall not be there after #1127 - it must be a leftover from a past merge
